### PR TITLE
Controller SIS/SUC Options

### PR DESCRIPTION
**Pull request by cdjackson**
Improves options for setting SUC/SIS
Signed-off-by: Chris Jackson <chris@cd-jackson.com>

### CODE REVIEW

**Comment by cdjackson** [comment]
#536

File: src/main/java/org/openhab/binding/zwave/internal/protocol/initialization/ZWaveNodeInitStageAdvancer.java [hunk 1]
```diff
@@ -277,7 +277,6 @@ private boolean processTransaction(ZWaveMessagePayloadTransaction transaction, l
                 try {
                     Thread.sleep(backoff);
                 } catch (InterruptedException e) {
-                    e.printStackTrace();
```
Comment (openhab-5iver) [inline src/main/java/org/openhab/binding/zwave/internal/protocol/initialization/ZWaveNodeInitStageAdvancer.java]: Should this have been replaced with a logger.error?

Comment (cdjackson) [reply src/main/java/org/openhab/binding/zwave/internal/protocol/initialization/ZWaveNodeInitStageAdvancer.java]: No.

**Comment by cdjackson** [comment]
This has been included in the 4th March test version, so merging.

**Pull request merged**